tests/BasicTest.php

```diff
@@ -62,7 +63,7 @@ public function testSimple2(): void
 
         $validator->rule('name', ['required', ['lengthMin', 3]]);
 
-        $err = $model->createEntity()->set('name', 'a')->validate();
+        $err = $model->createEntity()->setNull('name')->validate();
```

Comment:
why?

Reply:
Because we're testing "name required" rule here. lengthMin rule is tested in previous test testSimple1.